addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -113,7 +113,7 @@ def _get_livechat_mail_channel_vals(self, anonymous_name, operator, user_id=None
         visitor_user = False
         if user_id:
             visitor_user = self.env['res.users'].browse(user_id)
-            if visitor_user and visitor_user.active:  # valid session user (not public)
+            if visitor_user and visitor_user.active and visitor_user != operator:  # valid session user (not public)
```

Comment:
I can't reproduce the duplicate member issue in v13 actually, @oomsveta do you remember how we had duplicate member in guest PR when added the unique constraint?

Reply:
Or maybe I'm just checking it wrong? I was looking at the list of members on the form view, but maybe it was only an invisible issue in the database?

Reply:
Unfortunately, I only know the problem through your explanations, I've never experienced it myself.

I just tested it in 13.0, here are the steps I followed:
- Connect as Mitchell Admin
- Install livechat
- Open the livechat in a new tab (while being connected as Mitchell Admin) and send a message

Then I checked the database and noticed that there was only one `channel_partner` in the livechat channel, so there doesn't seem to be any duplicate data.

Reply:
Maybe add unique index on (channel_id, partner_id) on v13 and see if the issue raises in this case?
But otherwise I guess we can close the PR, and maybe check the same in v14 to be sure, and do nothing if no issue.
It might be possible the issue was introduced only in v15.

Reply:
Je vois pas bien comment reproduire en V14 non plus 
![image](https://user-images.githubusercontent.com/1810149/162903605-a2e4211b-7c26-40c8-a027-07b2af028d06.png)

Reply:
Livechat with yourself is extremely useful for testing the feature works, for us but also for clients with only one or few users.

For the delay I guess crosstab was even more broken in v13 than it is now, you should create a separate task to report the bug and investigate.

Duplicate issue not reproducible so closing this PR. I'll check in v14 also and close the task if necessary.